0.19/docker-entrypoint.sh

```diff
@@ -19,7 +17,7 @@ fi
 
 if [ "$1" = "bitcoind" ] || [ "$1" = "bitcoin-cli" ] || [ "$1" = "bitcoin-tx" ]; then
   echo
-  exec gosu bitcoin "$@"
```

Comment:
This way this check becomes useless, remove entirely being gosu not present anymore.